geckomat/limit_proteins/measureAbundance.m

```diff
@@ -1,11 +1,15 @@
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-% [f,count] = measureAbundance(enzymes)
+function [f,count] = measureAbundance(enzymes)
+% measureAbundance
+%
+% Computes the f factor, as a proxy to the mass fraction of proteins accounted 
+% for in an ecModel out of the total protein content in cells. An
+% integrated quantitative proteomics dataset from the https://pax-db.org/
+% database (stored in this toolbox as: 'GECKO/databases/prot_abundance.txt')
+% is used as a comparison basis.
 % 
-% Benjamin J. Sanchez. Last edited: 2018-08-10
-% Ivan Domenzain.      Last edited: 2018-11-26
```

Comment:
A side-comment: the `git blame` functionality, also available directly on GitHub ([example](https://github.com/SysBioChalmers/GECKO/blame/3f5f31660712276507acdc99ad3ebe740e73826e/geckomat/limit_proteins/measureAbundance.m)) is much more useful for authorship purposes, so I would encourage removing lines such as these from the entire codebase.

Reply:
True, doesn't make so much sense to keep this info in our scripts. Forgot to remove these lines in this PR, but it is probably good to have a PR specifically for doing it for all functions in the toolbox

Reply:
as discussed via email with @IVANDOMENZAIN, I'm planning a separate PR that among other things will remove these lines. As soon as we merge all the current PRs I will proceed with it.